src/Core/Help/Documentation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Help;
+
+use PrestaShop\PrestaShop\Core\Foundation\Version;
+
+class Documentation
+{
+    /**
+     * @var Version
+     */
+    private $version;
+
+    /**
+     * @var string
+     */
+    private $host;
+
+    public function __construct(Version $version, string $host)
+    {
+        $this->version = $version;
+        $this->host = $host;
+    }
+
+    public function generateLink(string $section, string $langIsoCode): string
```

Comment:
Hi @FabienPapet 

I just noticed this PR, and it made me realize that in one of my PRs (currently in QA) I introduced a provider to generate a doc link:
https://github.com/PrestaShop/PrestaShop/pull/28752/files#diff-4c74f03fdd9cfdda6a78c7fcd86670f9874ba0bb69133717c45facdf63e9f126R82

So I guess depending on which PR is merged first we'll have to adapt the other. I think my `HelpProvider` should either be replaced by your `Documentation` service, or maybe it should at least rely on it Because my service also handles a title and use an `admin_common_sidebar` internal route like it's done in the `FrameworkBundleAdminController` so this overlay is probably still relevant.

Reply:
I just saw your modif in `FrameworkBundleAdminController` which was the base of my code, so it seems I will have to use your `prestashop.core.help.documentation` service indeed